app/src/main/java/org/traccar/client/MainActivity.java

```diff
@@ -65,12 +68,12 @@
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-
         if (BuildConfig.HIDDEN_APP) {
             removeLauncherIcon();
         }
 
         sharedPreferences = PreferenceManager.getDefaultSharedPreferences(this);
+        migratePreferencesIfNecessary(sharedPreferences);
```

Comment:
I would rename it to something like `migrateLegacyPreferences` to make it more clear what it actually does.